src/lib/EventPublisher.js

```diff
@@ -77,7 +81,9 @@ class EventPublisher extends EventEmitter {
     eventData: EventData,
   ): Promise<Object> => {
     const eventID = uuid();
-    const requestEventName = `${getRequestEventName(eventData.name)}/${eventID}`;
+    const requestEventName = `${getRequestEventName(
```

Comment:
this formatting can't be prettier, I just tried to commit this line with last dev and it fixes to one line.